en-US/focus-ios.xliff

```diff
@@ -922,7 +927,7 @@
       </trans-unit>
       <trans-unit id="ShareMenu.SharePage">
         <source>Share Page With…</source>
-        <target>Share Page With…</target>
+        <target>Share Page With...</target>
```

Comment:
Interesting, we updated the String inside the project to use ellipsis https://github.com/mozilla-mobile/focus-ios/blob/ce23f59437e482de499a3fe26e3b1adc6ed35e56/Blockzilla/UIComponents/UIConstants.swift#L251 . But I'm not sure why it's exported like that. I assume when we will get the string update, it will be correctly formatted right?

Reply:
It really depends how you're importing back strings (the string was already fixed [3 months ago](https://github.com/mozilla-l10n/focusios-l10n/commit/a013f0d3a9b00698903455536a2a27bf0dd0a9fd) on our side.

I would just manually fix that file at this point.